apps/ideaspace/src/pages/workshop/[ideaId].js

```diff
@@ -1,7 +1,28 @@
 import Head from 'next/head';
 import WorkshoppingPage from '../../components/modules/WorkshoppingPage/WorkshoppingPage';
+import React from 'react';
+import { useRouter } from "next/router";
+
+export default function WorkshopIdeaPage() {
+  const router = useRouter();
+  const { idea } = router.query;
```

Comment:
is the `router` and `idea` being used for anything here?

Reply:
they aren't being used

Reply:
Can we remove it then?

Reply:
I removed it, a commit is coming soon